### ISSUE TITLE
Remove annoying printlns in `requireDependencies`

### ISSUE THREAD

**Issue by jvican** [portable-scala/sbt-crossproject#52]
Running the build with this plugin enabled outputs the following log:
```
Project defines platforms: jvm, js
project core
  
  discards: native
Project defines platforms: jvm, js
project core
  
  discards: native
Project defines platforms: jvm, js
project core
  
  discards: native
Project defines platforms: jvm, js
project core
  
  discards: native
Project defines platforms: jvm, js
project core
  
  discards: native
Project defines platforms: jvm, js
project core
  
  discards: native
project state
  
  discards: native
```
  
I think this is pretty annoying. IMO, the right way to do this is to use the sbt logger and adding a setting key to disable its output. The logic for this (`println`s) is in `requireDependencies`.

**Comment by densh** [comment]
Thanks for the suggestion!

**Comment by olafurpg** [comment]
Agreed, these spurious warnings appear in fastparse here https://github.com/lihaoyi/fastparse/pull/160 It's not strictly a "blocking" issue, but I feel bad contributing a fastparse port to native when this warning prints out on sbt load.

**Comment by jvican** [comment]
@sjrd explains how this can be addressed in a more principled way in #62.
